build.gradle

```diff
@@ -5,12 +5,15 @@ plugins {
   id 'jacoco'
   id 'io.spring.dependency-management' version '1.0.11.RELEASE'
   id 'org.springframework.boot' version '2.4.5'
-  id 'org.owasp.dependencycheck' version '6.2.1'
-  id 'com.github.ben-manes.versions' version '0.39.0'
```

Comment:
I think the com.github.ben-manes.versions needs adding back in. 

Looks like this is what provides the dependencyUpdates section (which was causing the build issue I'm getting)